conans/client/tools/files.py

```diff
@@ -304,7 +304,7 @@ def _path_equals(path1, path2):
     return path1 == path2
 
 
-def _valid_lib_extensions():
+def valid_lib_extensions():
```

Comment:
Agree with @jgsogo, better declare just a tuple, not a function

Reply:
done!